sds_data_manager/lambda_code/SDSCode/database/models.py

```diff
@@ -27,3 +22,84 @@ class UniversalSpinTable(Base):
     # TODO: create table for repointing and then make
     # a foreign key to universal_spin_table
     repointing_number = Column(Integer, nullable=False)
+
+
+class MetadataTable:
+    """Common Metadata table"""
+
+    id = Column(Integer, primary_key=True)
+    file_name = Column(String, nullable=False)
+    # TODO: Do we need instrument since these will be split up into their
+    # separate tables?
+    instrument = Column(String, nullable=False)
+    data_level = Column(String, nullable=False)
+    descriptor = Column(String, nullable=False)
+    start_date = Column(DateTime(timezone=True), nullable=False)
+    end_date = Column(DateTime(timezone=True), nullable=False)
+    ingestion_date = Column(DateTime(timezone=True), nullable=False)
+    version = Column(Integer, nullable=False)
+    format = Column(String, nullable=False)
```

Comment:
what is format? is it file extension? If so, can we change it to `file_extension` or something like that?

Reply:
I was torn on using `format` vs `extension` too. I'm happy to change the filename docs if we prefer `extension`